tst/DtlsFunctionalityTest.cpp

```diff
@@ -0,0 +1,169 @@
+#include "WebRTCClientTestFixture.h"
+
+namespace com {
+namespace amazonaws {
+namespace kinesis {
+namespace video {
+namespace webrtcclient {
+
+class DtlsFunctionalityTest : public WebRtcClientTestBase {
+  public:
+    STATUS createAndConnect(TIMER_QUEUE_HANDLE timerQueueHandle, PDtlsSession* ppClient, PDtlsSession* ppServer)
+    {
+        struct Context {
+            std::mutex mtx;
+            std::queue<std::vector<BYTE>> queue;
+        };
+        STATUS retStatus = STATUS_SUCCESS;
+        DtlsSessionCallbacks callbacks;
+        SIZE_T connectedCount = 0;
+        PDtlsSession pClient = NULL, pServer = NULL;
+        UINT64 sleepDelay = 20 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
+        Context clientCtx, serverCtx;
+
+        MEMSET(&callbacks, 0, SIZEOF(callbacks));
+        callbacks.stateChangeFn = [](UINT64 customData, RTC_DTLS_TRANSPORT_STATE state) {
+            if (state == CONNECTED) {
+                ATOMIC_INCREMENT((PSIZE_T) customData);
+            }
+        };
+        callbacks.stateChangeFnCustomData = (UINT64) &connectedCount;
+
+        DtlsSessionOutboundPacketFunc outboundPacketFn = [](UINT64 customData, PBYTE pData, UINT32 dataLen) {
+            Context* pCtx = (Context*) customData;
+            ASSERT_TRUE(pCtx != NULL);
+            ASSERT_TRUE(pData != NULL);
+            pCtx->mtx.lock();
+            pCtx->queue.push(std::vector<BYTE>(pData, pData + dataLen));
+            pCtx->mtx.unlock();
+        };
+
+        auto consumeMessages = [](Context* pCtx, PDtlsSession pPeer) -> STATUS {
+            STATUS retStatus = STATUS_SUCCESS;
+            ASSERT_TRUE(pCtx != NULL);
+            ASSERT_TRUE(pPeer != NULL);
+
+            pCtx->mtx.lock();
+            while (!pCtx->queue.empty()) {
+                auto& msg = pCtx->queue.front();
+                auto readLen = (INT32) msg.size();
+                CHK_STATUS(dtlsSessionProcessPacket(pPeer, (PBYTE) &msg.front(), &readLen));
+                pCtx->queue.pop();
+            }
+
+        CleanUp:
```

Comment:
I don't see any usage of this label any more. Some tools will complain

Reply:
The label is still being used here, https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/903/files#diff-ac5742313d7723b10c6452ef9d124727ed14d131e2dccbdfe676005259d9d113R52